website/src/server.mjs

```diff
@@ -50,6 +51,9 @@ export function makeServer({
             path.join(router.wwwRootPath, classifiedDirectory.path),
             { currentURI: request.path }
           );
+          const $ = cheerio.load(out);
+          $('script[id="emptyScript"]').remove();
```

Comment:
To find the empty elements, could we do `$('script')` then ask each element's `textContent` to find the empty elements? (Does Cheerio support `textContent` or something similar?)